app/services/github/repositories/sync.rb

```diff
@@ -32,6 +27,33 @@ def call
       def repositories
         @repositories ||= Repository.all
       end
+
+      def repository_owner_and_name(repository_link)
+        repository_link.split('/')[-2..-1]
+      end
+
+      def update_repository_languages(repository)
+        repository_owner, repository_name = repository_owner_and_name(repository.link)
+        request = client.repos.languages(repository_owner, repository_name)
+
+        if request.success?
+          languages = request.body.keys[0..MAX_LANGUAGES_NUMBER-1].join(', ')
+
+          repository.update!(language: languages)
+        end
+      end
+
+      def update_repository_stats(repository)
+        repository_owner, repository_name = repository_owner_and_name(repository.link)
+        request = client.repos.get(repository_owner, repository_name)
+
+        if request.success?
+          repository.update!(
+            issues: request.body['open_issues_count'],
+            stars: request.body['stargazers_count']
+          )
+        end
+      end
```

Comment:
```suggestion
      def update_repositoryrepository)
        repository_owner, repository_name = repository_owner_and_name(repository.link)
        request = client.repos.languages(repository_owner, repository_name)

        if request.success?
        languages = request.body.keys[0..MAX_LANGUAGES_NUMBER-1].join(', ')
        
          repository.update!(
            issues: request.body['open_issues_count'],
            stars: request.body['stargazers_count'],
            language: languages
          )
        end
      end
```

Since you have the same request with the same parameters and doing the same check, I think you can have just one single method `update_repository` and this would update all parameters at once. If you use this approach, I don't you need a `transaction` block btw. What do you think?